Handbook: Editorial Q3 2026 goals (#17862)
PostHog/posthog.com: contents/teams/editorial/objectives.mdx

## Business as usual

- A high-quality newsletter every two weeks - <TeamMember name="Ian Vanagas" photo />
- Social posts every day and amplifying employee accounts - <TeamMember name="Liam Graham" photo />

## Social handbook

**Owner:** <TeamMember name="Liam Graham" photo />

**Motivation:** Honeymoon period is almost over. I’ve gotta calcify a few ideas and best practices, and make sure they’re understood across the entire org.

**What we'll ship:** A handbook section about how we do social at PostHog. Answers how social posting fits into our broader goals, and what type of posts do well. Is prescriptive for the brand itself.

**We'll know we're successful when:** Everyone across PostHog understands our disposition as it regards social media, I can say “no” to a social idea by replying with :rtf-handbook:. Product marketers can write the first draft of social posts themselves. 

> ## Carousels

Reviewer: ```suggestion
## Test Instagram carousels
```

**Owner:** <TeamMember name="Liam Graham" photo />

**Motivation:** We want to expand the type of content we post to unlock Instagram as a channel for social posts. Carousels are the way to do this. 

**What we'll ship:** 

> 1. A graphics-approved template for Instagram carousels.

Reviewer: This is what we'll ship, not a measure of success.

2. The actual implementation of carousels to promote newsletters and blogs. 

> **We'll know we're successful when:** We've posted at least 6 carousels on Instagram. 

Reviewer: Why 6? This feels like a weird arbitrary number. @liamg-posthog – what's the ideal cadence for posts on Insta? Is it one of two carousels a week? Let's a pick a frequency, not a number of posts.

Reviewer 2: 6 was a vibed-out number of "how many of these would actually be really good as carousels so that we're focusing on quality, not quantity". I'd say once a week would work, if we wanted to tie it to a certain cadence.

## Expand AEO footprint (v2, the remix)

**Owner:** <TeamMember name="Natalia Amorim" photo />

**Motivation:** We've built a solid foundation - now it's time to go deeper and experiment with new content formats. The goal is to show up consistently when developers and founders are asking LLMs about the spaces PostHog competes in.

**What we'll ship:** 
- Continued optimization plus net-new AEO/SEO content to grow upper-funnel traffic (both internal and from freelancers)
- SEO positioning that backs the "Self-driving" push – updating website pages and making sure AEO/SEO content carries the message consistently
- Track self-driving (and related) prompts in Gauge

**We'll know we're successful when:** LLM traffic not only stays up and to the right, but picks up momentum

> ## Figure out X posting promoting content

Reviewer: In hindsight, I think we should drop this goal and @jina-yoon should focus on something else. Reasoning:

1. Ultimately, we hired Liam to own social. Doesn't make sense to hand this to someone else. In fact, newsletter distribution was an explicit goal of the role.

2. Where promoting content is concerned, I think we'd be better off getting better at something we already do – e.g. Ian's posting on LinkedIn – or leveraging James account, which I could work into what I want to do there.

3. This could easily just fall under Liam's business as usual posting.

Reviewer 3: that's valid, i'll come up something else - maybe either more on blogs or supporting more of the freelance editing. will update today

> ## Mini newsletters & blogs to explain trending dev topics

Reviewer 3: @ivanagas @andyvan-ph updated my main goal to this!

**Owner:** <TeamMember name="Jina Yoon" photo />

**Motivation:** Ian's loops article is [doing really well on Twitter](https://posthog.slack.com/archives/C09GU689J1X/p1782295022075899?thread_ts=1782289510.165929&cid=C09GU689J1X), and we're getting feedback that it's valuable to have short and simple explainers for overly-hyped and over-complicated topics. 

**What we'll ship:** A mini-newsletter or blog every two weeks explaining a trending topic, buzzword, or concept in simple terms. Aim for 2-minute reads with newsletter-level quality, written with the X and YC audience in mind. Example topics: agents, skills, orchestration 

> **We'll know we're successful when:** At least one of our explainers is near the top results when you search for it on Twitter + They get engagement on Twitter + People aren't unsubscribing from the newsletter due to feeling spammed. (SEO/AEO will likely benefit but that's not a primary goal.)

Author: Tweaking this one, but looks good @jina-yoon 

We don't really keep track of unsubscribes so that doesn't matter much.

```suggestion
**Motivation:** Ian's loops article [did really well on Twitter](https://posthog.slack.com/archives/C09GU689J1X/p1782295022075899?thread_ts=1782289510.165929&cid=C09GU689J1X) and we're getting feedback that it's valuable to have simple explainers for overly-hyped and over-complicated topics. 

**What we'll ship:** A mini-newsletter or blog every two weeks explaining a trending topic, buzzword, or concept in simple terms. Aim for 2-minute reads with newsletter-level quality, written with the X and YC audience in mind. Example topics: agents, skills, orchestration. 

**We'll know we're successful when:** They perform well on Twitter and people are referencing them as a great explainer on that topic. SEO/AEO will likely benefit but that's not a primary goal.
```

## A theory of self-driving

**Owner:** <TeamMember name="Ian Vanagas" photo />

> **Motivation:** We've agree that self-driving product is the most important concept for the next era of PostHog, but few people know about it and it's still unclear what the implications of self-driving products is. 

Reviewer: ```suggestion
**Motivation:** Self-driving product is the most important concept for the next era of PostHog, but few people know about it and it's still unclear what the implications of self-driving products is. 
```

**What we'll ship:** 

- 4 newsletters on an aspect of self driving product
- For example: setting good goals, doing better code reviews, getting the right context (using warehouse)

> **We'll know we're successful when:** We are known for "self-driving product," are more confident about what it means, and are having the term repeated by to us. 

Reviewer: ```suggestion
**We'll know we're successful when:** We are known for "self-driving product," are more confident about what it means, and are having the term repeated back to us. 
```

## A minimum viable brand study

> **Owner:** <TeamMember name="Natalia Amorim" photo />

Author: I could be the owner of this one if you want @natalia-amorim, up to you

Reviewer: Related to my above comment, perhaps @jina-yoon should own this, or do so jointly with Nat? It's a newsletter thing ultimately, and I think this is a better goal for Jina than working on social stuff.

Reviewer 4: Framing this as a newsletter-first asset is a bit of a departure from our "this can just be a graph and a few interesting stats and a couple words" MVP chat because we have a higher bar for the newsletter, so it makes for a very different (and much more rigorous/long) process etc etc 

So if that's the angle happy to pass the torch to Jina or Ian. 

That said from the POV of launching a _very simple_ MVP I will probably try to take a stab at this as a blog post/simpler asset for AEO, but I don't need to own the goal or make it a priority

Reviewer: I don't think we need to have a higher bar to put this in the newsletter, provided the insights are interesting. if it's just a list of 6 data points on a related theme, some light commentary + visuals, that's absolutely fine and would work in the newsletter.

Reviewer 4: SGTM! @jina-yoon @ivanagas will leave it up to you to edit the owner here & approve the PR on my end

Reviewer 3: i decided not to take this on - personally not excited enough to drive it, but happy to help with editing if you need it @natalia-amorim! leaving you as the owner

Reviewer 4: @ivanagas since this is newsletter should we re-assign to you?

Author: No, let's have you own it and aim for it to be a newsletter as Andy says, but if it falls to being a blog, that's ok. I'm happy to help.
